esphome/components/wifi/__init__.py

```diff
@@ -345,6 +350,10 @@ async def to_code(config):
 
     cg.add_define("USE_WIFI")
 
+    if config.get(CONF_IGNORE_EFUSE_MAC_CRC, False):
+        cg.add_define("USE_IGNORE_EFUSE_MAC_CRC")
```

Comment:
```suggestion
        cg.add_define("USE_ESP32_IGNORE_EFUSE_MAC_CRC")
```

With the above, then let's also rename the define to `USE_ESP32_IGNORE_EFUSE_MAC_CRC` (so that the USE shows which module it's from)

Reply:
Done.